app/models/ItemLevelDocuments.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import config.FrontendAppConfig
+import services.DocumentsService
+
+case class ItemLevelDocuments(
+  previous: Int,
+  supporting: Int,
+  transport: Int
+) {
+
+  def canAdd(documentType: String)(implicit config: FrontendAppConfig): Boolean = documentType match {
```

Comment:
Might be worth introducing a DocumentType trait so we don't have these strings everywhere